draft-birkholz-rats-epoch-markers.md

```diff
@@ -229,11 +235,11 @@ Timestamp:
 time.  It MUST use the int format.
 
 Pad:
-: pad bytes used to make the stateless nonce the desired size.
+: zero or more pad bytes, used to make the stateless nonce the desired size.
 
 AuthTag:
-: HMAC w/ SHA-1 computed over the CBOR serialisation of TimeToken encoded as a
-20-bytes string.
+: HMAC w/ SHA-224 computed over the CBOR serialisation of TimeToken encoded as
+a 28-bytes string.
```

Comment:
Now if we could spend four more bytes to get a hash that is actually widely supported in hardware and software (SHA-256)?

Reply:
I guess we could.  The whole idea behind SHA-1 was being able to do everything in 32 bytes, which fits well in "a certain RoT"'s API.